app/src/main/resources/application-dev.properties

```diff
@@ -1,6 +1,6 @@
 #LOGGING
-%dev.quarkus.log.level=${LOG_LEVEL:INFO}
-%dev.quarkus.log.category."io.apicurio".level=${REGISTRY_LOG_LEVEL:DEBUG}
```

Comment:
We might consider an `apicurio.log.level` property to make this easier.  wdyt?

Like this maybe:

```
%dev.apicurio.log.level=DEBUG
%prod.apicurio.log.level=INFO
quarkus.log.category."io.apicurio".level=apicurio.log.level
```

Reply:
I don't think that improves the situation. If you want to change the value, you'll have two ways of configuring the same thing, $APICURIO_LOG_LEVEL and $QUARKUS_LOG_CATEGORY_IO_APICURIO_LEVEL, as we discussed, I went with just having the last one available, defaulting to Quarkus patterns instead of building our own ones.

Reply:
I agree with Eric that it makes sense as an exception. The `io.apicurio` is kind of exposing the internals of the application, and can possibly change in the future. But I won't insist.

Reply:
Well, this is not a general exception. The only reason for this property to be defined at the application-properties level is that, for the dev profile, that is not used when the application is running in production, we're changing the log level. In the prod mode, the Quarkus and Apicurio levels are just the same. I'll do another round removing properties from the different files since there is no need to prefix them with the profile: https://quarkus.io/guides/config-reference#profile-aware-files